### ISSUE TITLE
Pretty Close

### ISSUE THREAD

**Issue by lilbud** [RetroHursty69/es-theme-lightswitch#1]
Hello RetroHursty69,

I really don't like how similar your switch theme is to mine. I mean it's different in its own way but I feel its too close to mine. 

Sincerely,
Lilbud

**Comment by RetroHursty69** [comment]
Hi Lilbud

Wow, there is a Switch Theme already. Damn. To be honest, I don't play games, I don't use ES. I just think up ideas and make themes. I have to admit I do not even look to see what other people have made. Um, sorry mate, if I have offended you I assure you it was not intentional.

So you have a Switch Theme on ES already?

Regards
Dwayne

**Comment by RetroHursty69** [comment]
Hi Lilbud

I am really, really sorry. I had no idea you had made a Switch Theme already. I just went and had a look. I am not sure if you believe me or not but I honestly had no idea. One of the reasons I did this theme was because I did not think it had been done before. If I knew you had, I would not have done it. My main goal is to try and do new ideas when I make themes.

So I am really sorry this has happened. In future I will now make sure I look at the themes available in RetroPie before I proceed.

Again I am really sorry for any anger or frustration I have caused you.

Regards
Dwayne
